### ISSUE TITLE
SelectOneMenu: aria required not generated.

### ISSUE THREAD

**Issue by twuillemin** [primefaces/primefaces#14880]
### Describe the bug

**Issue**
When a SelectOneMenu has the attribute required="true", I expect it to be styled as required. However, the HTML "aria-required" is not generated anymore since version 5.0.13

**Possible reason**
While fixing the issue #14625 (SelectOneMenu: Missing aria-disabled attribute when disabled="true") the file primefaces/src/main/java/org/primefaces/component/selectonemenu/SelectOneMenuRenderer.java was changed, and the call for `renderAccessibilityAttributes(context, menu);` was removed in favor of a direct generation af the disable attibute with `writer.writeAttribute(HTML.ARIA_DISABLED, "true", null);`

However, the function `renderAccessibilityAttributes` is also in charge of generating other aria attribute such as... required.

### Reproducer

The following code works in 5.0.12 but not in 5.0.13 and onward
```
<p:selectOneMenu id="required" value="#{selectOneMenuView.selectedOption}" required="true">
    <f:selectItem itemLabel="Select One" itemValue=""/>
    <f:selectItem itemLabel="Option1" itemValue="Option1"/>
    <f:selectItem itemLabel="Option2" itemValue="Option2"/>
    <f:selectItem itemLabel="Option3" itemValue="Option3"/>
</p:selectOneMenu>
```

### Expected behavior

I expect a required SlectOneMenu to be styled as required when marked as required

### PrimeFaces edition

None

### PrimeFaces version

5.0.13

### JSF implementation

All

### JSF version

4.x

### Java version

21.x

### Browser(s)

Tested on FireFox 150 and last Safari

**Comment by twuillemin** [comment]
That was fast.

Thanks a lot for your hard work.

**Comment by melloware** [comment]
were you able to retest 15.0.15?

**Comment by twuillemin** [comment]
Good morning,

I just tested 15.0.5  and it works perfectly fine!
